Yarn install is skipped in gitlab (with 5.5.0)

<!--
- Please follow the issue template below for bug reports.
- If you have a support request rather than a bug, please use [Stack Overflow](http://stackoverflow.com/questions/tagged/jhipster) with the JHipster tag.
- For bug reports it is mandatory to run the command `jhipster info` in your project's root folder, and paste the result here.
- Tickets opened without any of these pieces of information will be **closed** without any explanation.
-->

##### **Overview of the issue**

Yarn install is skipped in gitlab ci and dependencies are not installed.
```
Running with gitlab-runner 11.4.0-rc1 (1ff344e1)
  on docker-auto-scale ed2dce3a
Using Docker executor with image jhipster/jhipster:v5.5.0 ...
Pulling docker image jhipster/jhipster:v5.5.0 ...
Using docker image sha256:8340a3f99511000b1424af4da1341a18603d9cad1e0e69ef232d5295b97cfe6a for jhipster/jhipster:v5.5.0 ...
Running on runner-ed2dce3a-project-8975696-concurrent-0 via runner-ed2dce3a-srm-1540144153-bfa9144c...
Cloning repository...
Cloning into '/builds/mikke/uustesmi'...
Checking out 84f70af0 as master...
Skipping Git submodules setup
Checking cache for master...
FATAL: file does not exist                         
Failed to extract cache
$ export GRADLE_USER_HOME=`pwd`/.gradle
$ ./gradlew yarn_install -PnodeInstall --no-daemon
Downloading https://services.gradle.org/distributions/gradle-4.10.2-bin.zip
..........................................................................

Welcome to Gradle 4.10.2!

Here are the highlights of this release:
 - Incremental Java compilation by default
 - Periodic Gradle caches cleanup
 - Gradle Kotlin DSL 1.0-RC6
 - Nested included builds
 - SNAPSHOT plugin versions in the `plugins {}` block

For more details see https://docs.gradle.org/4.10.2/release-notes.html

> Task :nodeSetup

> Task :yarnSetup
/builds/mikke/uustesmi/.gradle/yarn/yarn-v1.10.1/bin/yarn -> /builds/mikke/uustesmi/.gradle/yarn/yarn-v1.10.1/lib/node_modules/yarn/bin/yarn.js
/builds/mikke/uustesmi/.gradle/yarn/yarn-v1.10.1/bin/yarnpkg -> /builds/mikke/uustesmi/.gradle/yarn/yarn-v1.10.1/lib/node_modules/yarn/bin/yarn.js
+ yarn@1.10.1
added 1 package in 0.547s

> Task :yarn_install SKIPPED

BUILD SUCCESSFUL in 51s
2 actionable tasks: 2 executed
$ ./gradlew compileJava -x check -PnodeInstall --no-daemon
> Task :cleanResources UP-TO-DATE
> Task :bootBuildInfo
> Task :nodeSetup UP-TO-DATE
> Task :yarnSetup UP-TO-DATE
> Task :yarn_install SKIPPED

> Task :webpackBuildDev FAILED
yarn run v1.10.1
$ yarn run cleanup && yarn run webpack:build:main
$ rimraf build/www
/bin/sh: 1: rimraf: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':webpackBuildDev'.
> Process 'command '/builds/mikke/uustesmi/.gradle/yarn/yarn-v1.10.1/bin/yarn'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s
5 actionable tasks: 2 executed, 3 up-to-date
ERROR: Job failed: exit code 1
```

##### **Motivation for or Use Case**

Updated the application for 5.5.0 and now gitlab builds are failing without adding manual yarn command to gitlab configuration.

##### **Reproduce the error**

1. Create monolith application with yarn  and gradle
2. Generate gitlab configuration with `jhipster ci-cd`
3. Push repo to gitlab

##### **Related issues**

Not found

##### **JHipster Version(s)**

5.5.0. Worked at least with 5.1.0

##### **JHipster configuration**

Reproduces with generated application without changes

-   [x] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/ci-cd/templates/.gitlab-ci.yml.ejs
+++ b/generators/ci-cd/templates/.gitlab-ci.yml.ejs
@@ -47,7 +47,7 @@ before_script:
         <%_ if (clientPackageManager === 'yarn') { _%>
     - ./gradlew yarn_install -PnodeInstall --no-daemon
         <%_ } else if (clientPackageManager === 'npm') { _%>
-    - ./gradlew npmInstall -PnodeInstall --no-daemon
+    - ./gradlew npm_install -PnodeInstall --no-daemon
         <%_ } _%>
     <%_ } _%>
 

--- a/generators/server/templates/gradle/profile_dev.gradle.ejs
+++ b/generators/server/templates/gradle/profile_dev.gradle.ejs
@@ -85,10 +85,4 @@ processResources {
 processResources.dependsOn webpackBuildDev
 copyIntoStatic.dependsOn processResources
 bootJar.dependsOn copyIntoStatic
-
-<%= clientPackageManager %>_install.onlyIf { shouldWebpackRun() == true }
-
-def shouldWebpackRun() {
-    project.hasProperty('webpack')
-}
 <%_ } _%>

